tests/e2e/e2e.go

```diff
@@ -167,10 +167,7 @@ func (te *TestEnvironment) StartCluster() error {
 		}
 		tests.Outf("{{green}}successfully started network-runner: {{/}} %+v\n", resp.ClusterInfo.NodeNames)
 
-		// start is async, so wait some time for cluster health
-		time.Sleep(time.Minute)
```

Comment:
Was this sleep just never needed? Like a minute long sleep is _long_

Reply:
Not neede anymore

Reply:
The missing piece here is understanding that `Health` actually blocks until `Healthy`.

Reply:
yup that was very old ANR tech dep. with the first grpc server implementation, start (with blockchain creation -I know this is not the case but this is a copy probably-) broked posterior health call without sleeping some time.